README.adoc

```diff
@@ -19,12 +19,12 @@ A widget property could be bind with one ui-designer variable by one of this bin
 | interpolation
```

Comment:
Is there a reason to have the Readme in the asciidoc format?
This is not usual for a Readme, and one may have difficulties to read it in their IDE if the asciidoc plugin is not installed.

Reply:
Juste because i prefer ascidoc to markdown :D